sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1166,15 +1166,15 @@ def test_gradient_boosting_early_stopping():
     X, y = make_classification(n_samples=1000, random_state=0)
 
     gbc = GradientBoostingClassifier(
-        n_estimators=1000,
+        n_estimators=100,
         n_iter_no_change=10,
         learning_rate=0.1,
         max_depth=3,
         random_state=42,
     )
 
     gbr = GradientBoostingRegressor(
-        n_estimators=1000,
+        n_estimators=100,
```

Comment:
Same remark here for `gbr` (13 or 28 trees depending on the learning rate). Let's also revert:

```suggestion
        n_estimators=1000,
```